protocol/network/messages/verack.md

```diff
@@ -7,16 +7,6 @@ There is no version negotiation functionality between nodes; therefore if the no
 
 This `verack` message consists of only a message header; the command string is "verack".
 
-## Example Serialized Data
+## Message Format
 
-Net Magic<sup>[(BE)](/protocol/misc/endian/little)</sup>
```

Comment:
This got moved into the main messages page, as a one-off example since I figured we probably weren't going to do this with every type of message (and it's nice to have a simple example there regardless of whether we potentially add that in the future).